### DISCUSSION TITLE
plugin enabled but not running

### DISCUSSION THREAD

**Discussion by Drumsid** [rabbitmq/rabbitmq-delayed-message-exchange#208] [General]
Already everywhere asked, nowhere can give an answer. On my question.
I turn on the plugin according to the instructions, but for some reason it does not start. But only as a set. How to make it enabled? And appeared * near the letter E?
![notRunning](https://user-images.githubusercontent.com/45962995/186901961-5fec99f2-c4b6-4af7-9ff3-8a24cbee52c8.png)

Because of this, I can't create an "x-delayed-message" queue either in code or through the web interface.
I just need help on how to run the plugin, I'm not getting a clear answer anywhere yet

**Comment by michaelklishin** [comment]
See server logs for clues.

**Reply by Drumsid** [reply]
I try to look for logs using the **rabbitmq-diagnostics -q log_location** command, but the file pointed to by the command is empty. Or am I looking in the wrong place? Can you tell me where to look for the logs?

**Reply by michaelklishin** [reply]
It probably means that the node logs to standard streams

**Comment by Drumsid** [comment]
found such errors in docker container during startup.
`=ERROR REPORT==== 29-Aug-2022::13:53:03.758760 ===
    label: {erl_prim_loader,file_error}
    report: File operation error: eacces. Target: /path_to/rabbitmq_delayed_message_exchange-3.10.2.ez/ebin. Function: read_file_info. Process: code_server.
`
and a little lower

`Problem reading some plugins: [{"/opt/rabbitmq/plugins/rabbitmq_delayed_message_exchange-3.10.2.ez",
                                {invalid_ez,eacces}}]
`
This is clearly related to my problem, but it doesn't tell me anything. Can you suggest something based on this?

**Comment by Drumsid** [comment]
after this command
```
root@dev-rabbit-example:/# rabbitmq-plugins enable rabbitmq_delayed_message_exchange
Enabling plugins on node rabbit@dev-rabbit-example:
rabbitmq_delayed_message_exchange
The following plugins have been configured:
  rabbitmq_delayed_message_exchange
  rabbitmq_management
  rabbitmq_management_agent
  rabbitmq_prometheus
  rabbitmq_web_dispatch
Applying plugin configuration to rabbit@dev-rabbit-example...
Plugin configuration unchanged.
```
I see errors like this.

```
Problem reading some plugins: [{"/opt/rabbitmq/plugins/rabbitmq_delayed_message_exchange-3.10.2.ez",
                                {invalid_ez,eacces}}]
Problem reading some plugins: [{"/opt/rabbitmq/plugins/rabbitmq_delayed_message_exchange-3.10.2.ez",
                                {invalid_ez,eacces}}]
Problem reading some plugins: [{"/opt/rabbitmq/plugins/rabbitmq_delayed_message_exchange-3.10.2.ez",
                                {invalid_ez,eacces}}]

```

**Reply by michaelklishin** [reply]
`eacces` means the plugin file cannot be read by the node. Check the file permissions.

**Comment by Drumsid** [comment]
Thank you very much everything worked out!
